src/Events/Authorization/XacmlMapperHelper.cs

```diff
@@ -50,5 +50,24 @@ public static XacmlJsonCategory CreateSubjectAttributes(string subject)
 
             return category;
         }
+
+        /// <summary>
+        /// Splits the resource attribute at the final ':'
+        /// </summary>
+        /// <param name="resource">The resource string</param>
+        /// <returns>A string array with two items</returns>
+        /// <remarks>
+        /// First entry should be used as the attribute id in the xacml request
+        /// Second entry should be used as the atttribute value in the xaml request
+        /// For an Altinn App resource second entry is on format altinnapp.{org}.{app}
+        /// </remarks>
+        public static string[] SplitResourceInTwoParts(string resource)
```

Comment:
I don't like tuples, but they are better than an array. Use the stack and not the heap!